inst/shiny/ui_04_diffex.R

```diff
@@ -27,32 +27,33 @@ shiny_panel_diffex <- fluidPage(
           8,
           tabsetPanel(
             id = 'dataset',
-            tabPanel('Heatmap', plotOutput("diffPlot")),
-            tabPanel('Heatmap Options', 
-              fluidPage(
-                fluidRow(
-                  column(
-                    4,
-                    wellPanel(
-                      "General Options",
-                      checkboxInput("displayHeatmapRowLabels", "Display Row Labels", value=TRUE),
-                      checkboxInput("displayHeatmapColumnLabels", "Display Column Labels", value=TRUE),
-                      checkboxInput("displayHeatmapColumnDendrograms", "Display Column Dendrograms", value=TRUE),
-                      checkboxInput("displayHeatmapRowDendrograms", "Display Row Dendrograms", value=TRUE),
-                      textInput("heatmapColumnsTitle", "Columns Title", value = "Differential Expression")
-                    )
-                  ),
-                  column(
-                    4,
-                    wellPanel(
-                      "Colorbar Options",
-                      checkboxInput("displayHeatmapColorBar", "Color Bar", value=TRUE),
-                      uiOutput("colorBarOptions")
-                    )
-                  )
-                )
-              )
-            ),
+            tabPanel('Heatmap', 
+                     fluidPage(
+                       fluidRow(
+                         column(
+                           4,
+                           br(),
+                           wellPanel(
+                             "General Options",
+                             checkboxInput("displayHeatmapRowLabels", "Display Row Labels", value=TRUE),
+                             checkboxInput("displayHeatmapColumnLabels", "Display Column Labels", value=TRUE),
+                             checkboxInput("displayHeatmapColumnDendrograms", "Display Column Dendrograms", value=TRUE),
+                             checkboxInput("displayHeatmapRowDendrograms", "Display Row Dendrograms", value=TRUE),
+                             textInput("heatmapColumnsTitle", "Columns Title", value = "Differential Expression")
+                           ),
+                           wellPanel(
+                             "Colorbar Options",
```

Comment:
Putting these options alongside the heatmap makes the heatmap pretty squished. Can you put the options on a tabsetpanel inside the heatmap tab? I think that might be a good solution.